dias/utils/tracker.py

```diff
@@ -157,6 +157,10 @@ class Tracker:
     ----------
     path : String
         the location of the staging directory that the user is interested in
+    file_tracking_db : String
+        the location of the file index database that the tracker is referencing
+    write : Boolean
+        flips between read_and_write and read_only mode. True is read_and_write, and is the default.
```

Comment:
Shouldn't the default be read only?

Reply:
btw I think the heading for this list should be `Parameters`:
https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
Read-only default makes a lot of sense! I chose read_and_write as the default because that meant updating fewer things. =) But it makes sense enough that I should bite the bullet.